src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -30,7 +30,7 @@ export const getManifestResource = (
 }
 
 export const getDesiredManifestResource = (appDetails: AppDetails, podName: string, nodeType: string) => {
-    const selectedResource = appDetails.resourceTree.nodes.filter(
+    const selectedResource = appDetails.resourceTree?.nodes.filter(
```

Comment:
the value will be undefined, is there any handling further in the code

Reply:
Yes